docker/Dockerfile

```diff
@@ -32,6 +32,18 @@ ADD / /opt/gcp_variant_transforms/src/
 # Needed for installing mmh3 (one of the required packages in setup.py).
 RUN apt install -y g++
 
+# Install Pysam dependencies.
```

Comment:
I think it's worth calling out that this is only needed because we have a monolithic binary.  Otherwise we only need these dependencies on the Dataflow worker VM.